src/popup/components/Modals/TransferReceive.vue

```diff
@@ -45,18 +45,25 @@
       </div>
 
       <div class="request-specific-amount">
-        <InputAmount
+        <Field
```

Comment:
Why do we need to wrap it with Field now?

Reply:
`VeeValidate` has completely changed since v2. The `v-validate` property doesn't exist anymore, now you need to pass the rules to this new `Field` component which handles the validation